### DISCUSSION TITLE
create view配上regexp_extract的奇怪表现

### DISCUSSION THREAD

**Discussion by howardyan93** [apache/doris#26620] [A - General / Q&A]
`select field1, regex_extract(field1, pattern, 1) result1 from table1;`

有结果。
但是当我把它创建为**view**时

`create view test as select field1, regex_extract(field1, pattern, 1) result1 from table1;`

再运行

`select field1, result1 from test;`

**result1**结果就是空字符串。

感觉跟我预期的不太一样。正则的regexp_extract只能在select里面使用么？

**Comment by LemonLiTree** [comment]
which doris version？
can you provide table1  creation statement？

**Reply by howardyan93** [reply]
doris 2.0.2。
我在测试环境里试了一下，小数据量不会有这个问题。数据量上亿才会出现。
